tests/testCoaddInputs.py

```diff
@@ -207,6 +207,7 @@ def getCoaddPath(self, version):
         return os.path.join(self.dataDir,
                             "testCoaddInputs_coadd_with_version_{}_data.fits".format(version))
 
+    @unittest.skip("Depends on BaseRecord get support, which will be changed in DM-8716.")
```

Comment:
I'm quite worried about this. I thought we had already properly wrapped BaseRecord.get.

I filed DM-8716 as a request to clean up ugly code, not to add functionality. (I am also hoping that `get` and `__getitem__` can be made do the same thing, instead of almost the same thing. But the small differences are inherited from SWIG)